auth/postgres/init.go

```diff
@@ -51,6 +51,14 @@ func Migration() *migrate.MemoryMigrationSource {
 					`DROP TABLE IF EXISTS keys`,
 				},
 			},
+			{
+				Id: "auth_2",
+				Up: []string{
+					`ALTER TABLE domains 
+                     ALTER COLUMN alias SET NOT NULL, 
+                     ADD CONSTRAINT alias_not_empty CHECK (alias <> '');`,
```

Comment:
there is a need for this constraint. It makes sense, but for consistency with the rest of the codebase, let's have this check on API or svc layers.

Reply:
Since you have added API validation, please remove it from the DB for the sake of consistency with the rest of the codebase.

Reply:
Okay